### PR TITLE
add: favicon to head in template

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Added favicon to the i18n tracker

#### Related issues & labels (optional)

- Suggested label: site improvement

### CODE REVIEW

File: scripts/lib/translation-status/template.html [hunk 1]
```diff
@@ -12,6 +12,8 @@
 			content="Translation progress tracker for the Astro Docs site. See how much has been translated in your language and get involved!"
 		/>
 		<link rel="canonical" href="https://i18n.docs.astro.build/" />
+		<link rel="icon" href="/favicon.ico" sizes="32x32" />
+		<link rel="shortcut icon" href="/favicon.svg" type="image/svg+xml">
```
Comment (HiDeoo) [inline scripts/lib/translation-status/template.html:16]: Thanks for the contribution 🙌 

Small question: I tested locally and I did not get any favicon. Where are the assets supposed to come from? Should they also be part of the PR?

Comment (dreyfus92) [reply scripts/lib/translation-status/template.html:16]: They are supposed to be coming from public, but apparently they are not being loaded. I don't know if the `href` should be replaced with something like `https://docs.astro.build/favicon.ico`.

Comment (delucis) [reply scripts/lib/translation-status/template.html:16]: The i18n dashboard is not built with Astro so doesn’t use `public/`. Given we’ll soon switch to [Lunaria](https://lunaria.dev/), maybe this is something to fix there if @Yan-Thomas hasn’t already considered it?

Comment (delucis) [reply scripts/lib/translation-status/template.html:16]: > I've considered and was about to implement inline SVG favicon support, you can see more about this method here: [austingil.com/svg-favicons](https://austingil.com/svg-favicons/)

TIL!

> I can just replace with what I just previously mentioned, it works tho. And maybe later patch it when Lunaria has figured out the best way to tackle this.

@dreyfus92 that makes sense! Yeah we can just point at the docs.astro.build version for now 👍

Comment (dreyfus92) [reply scripts/lib/translation-status/template.html:16]: @delucis cool, cool. I will make the fix rn.

**Review by ElianCodes — APPROVED** [review]
Looks like it does indeed work now!

**Pull request merged**